src/k-provider/ovp/external-captions-builder.js

```diff
@@ -13,14 +13,15 @@ const CaptionsFormatsMap: {[format: string]: string} = {
 };
 
 class ExternalCaptionsBuilder {
-  static createConfig(captions: Array<Object>): Array<PKExternalCaptionObject> {
+  static createConfig(captions: Array<Object>, ks: string): Array<PKExternalCaptionObject> {
```

Comment:
Add an optional annotation

Reply:
it's not optional, it'll pass empty if it's not exist